ocs_samples/basic_samples/SDS/DotNet/SdsRestApiCore/SdsRestApiCore/Program.cs

```diff
@@ -62,7 +62,7 @@ public static async Task<bool> MainAsync(bool test = false)
             Console.WriteLine(@"        \/      \/    \/        \/        \/        \/            ");
```

Comment:
please remove lines 21-23, same as the other one
also bunch of empty lines in this file if you don't mind cleaning those up